test/parallel/test-process-startTime.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+
+const now = Date.now();
+const diffFromNow = now - process.startTime;
+
+// Make it a bit flexible
+assert.ok(diffFromNow > 0);
```

Comment:
Wouldn't want this one removed. Just want to make sure `>` is the right operator and not `>=`, that's all.